src/partials/welcome.ts

```diff
@@ -262,18 +263,33 @@ class WelcomeController {
             resume: false,
         });
 
-        // Set up the broadcast channel that checks whether we're already connected in another tab
-        this.setupBroadcastChannel(this.webClientService.salty.keyStore.publicKeyHex);
-
         // Initialize QR code params
         this.$scope.$watch(() => this.password, () => {
             const payload = this.webClientService.buildQrCodePayload(this.password.length > 0);
             this.qrCode = this.buildQrCode(payload);
             this.passwordStrength = scorePassword(this.password);
         });
 
-        // Start webclient
-        this.start();
+        // Set up the broadcast channel that checks whether we're already connected in another tab
+        this.setupBroadcastChannel(this.webClientService.salty.keyStore.publicKeyHex, 0)
```

Comment:
Why a timeout of 0? Won't this immediately resolve to `no_answer`?

Reply:
Yup. Should be valid for the *scan* case.

Reply:
Ah, I see, in this case we set up the broadcast channel so it can answer requests from other tabs, but we don't expect a valid answer ourselves (since this is a new connection) :slightly_smiling_face: